src/Monolog/Handler/ElasticsearchHandler.php

```diff
@@ -69,6 +81,14 @@ public function __construct(Client $client, array $options = [], $level = Logger
             ],
             $options
         );
+
+        if ($client instanceof Client8 || $client::VERSION[0] === '7') {
```

Comment:
When is this comparison false? Clients before version 7 are not supported

Reply:
Well only those versions are used in the tests but we don't enforce it when used within a project so I thought it safer to check and then used the same check in tests for consistency even tho there it's useless.